clang/tools/dpct/cmake/dpct.cmake

```diff
@@ -119,4 +119,9 @@ set(CMAKE_SYCL_FLAGS "")
 set(COMPATIBILITY_VERSION 99.9)
 set(COMPATIBILITY_VERSION_MAJOR 99)
 set(COMPATIBILITY_VERSION_MINOR 9)
-set(COMPATIBILITY_VERSION_STRING "${COMPATIBILITY_VERSION}")
\ No newline at end of file
+set(COMPATIBILITY_VERSION_STRING "${COMPATIBILITY_VERSION}")
+set(SYCL_TOOLKIT_ROOT_DIR "${ONEAPI_ROOT}")
+set(SYCL_TOOLKIT_INCLUDE "${ONEAPI_ROOT}/vtune/latest/include;${ONEAPI_ROOT}/vtune/latest/sdk/include")
+# OneAPI library is marked for linker lookup by FindIntelSYCL.cmake. So we can
+# safely set the CUDAToolkit_LIBRARY_DIR to ""
```

Comment:
1. According to the definition of CUDAToolkit_LIBRARY_DIR in https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html:
```
The path to the CUDA Toolkit library directory that contains the CUDA Runtime library cudart.
```
While in the SYCL side, the SYCL runtime library is auto-loaded when option "-fsycl" is specified, that is why we can set the helper variable SYCLToolkit_LIBRARY_DIR to ""
2.  There is no file "FindIntelSYCL.cmake" in oneAPI package, pls provide the correct file name you refer to.

Reply:
Done